pkgs/development/python-modules/pycm/default.nix

```diff
@@ -20,6 +20,8 @@ buildPythonPackage rec {
   '';
 
   checkInputs = [ pytest ];
+
+  buildInputs = [ matplotlib seaborn ];
```

Comment:
Should be propagated? these are technically runtime deps I suppose if you try to call `cm.plot()` without either of these libraries you'll get an error from pycm telling you to `pip install matplotlib`

Reply:
python packages should be propagated, yes.

Reply:
thanks, fixed.